src/modules/InternetDevices.rb

```diff
@@ -1,116 +0,0 @@
-# encoding: utf-8
```

Comment:
It appears unused in the control files so we can probably drop that too and cross reference the PR here.